### ISSUE TITLE
Fan Control has incorrect fan speeds

### ISSUE THREAD

**Issue by CrackHorse** [Rem0o/FanControl.Releases#2179]
I have 8 fans: 5 box, 1 CPU and 2 GPU. The GPU and 4 of the box fans work fine, but my CPU fan and one of my box fans run at a ridiculously slow rate.

The box fan can run ~1700 rpm normally, but Fan control thinks that 500 rpm is '100%'. Similarly, my CPU fan, which probably runs up to ~1500 rpm is at '100%' when it hits 300 rpm using fan control. I just got it a few weeks ago, and so far like it quite a bit, but if I can't run my CPU fan over 300 rpm, it's really not usable.

Any guesses as to how I tell Fan Control the correct max speeds?

If I turn off the fans within Fan Control and let other applications control them, then they go to their normal speeds and Fan Control shows the correct speed. So it will show the CPU fan at 1000+ rpm, but as soon as I enable it, back down to 300.

I have a z690-A Motherboard with a 12700k CPU.

**Comment by Rem0o** [comment]
Asus board right?  

I believe Asus "modes" in Armory Crate/their software, like "silent" and so on,  will artificially restrict the available range, hence achieving this effect. 

See https://github.com/Rem0o/FanControl.Releases/issues/1286

**Comment by CrackHorse** [comment]
Yes, it's ASUS.

Looks like I figured it out (in case anyone else runs into the same problem). The answer was to 'install' Armoury Crate, then go to fan Xpert and disable it. I'm guessing I still had it enabled when I had previously uninstalled Armoury Crate and somehow the zombie software was still controlling 2 fans.

I hadn't previously been using the modes (so it was the silent or other settings in my case; had previously been set to manual curves) and I had disabled everything in BIOS (although I think it was all monitoring) but that didn't help.
